rdr_service/services/ghost_check_service.py

```diff
@@ -59,12 +67,25 @@ def run_ghost_check(self, start_date: date, end_date: date = None):
             # Individually check each participant not seen in the API data yet (their date might be a little off).
             response = client.get_participant_lookup(participant_id=participant_id)
             is_ghost_response = response is None
-            self._record_ghost_result(
+            ghost_change = self._record_ghost_result(
                 is_ghost_response=is_ghost_response,
                 participant=participant_map[participant_id]
             )
+            if ghost_change:
+                pdr_rebuild_list.append({'pid': participant_id})
 
-    def _record_ghost_result(self, is_ghost_response: bool, participant: Participant):
+        task = GCPCloudTask()
+        for batch in list_chunks(pdr_rebuild_list, 100):
+            # PDR/BQ module views select the test(/ghost) setting from the participant data; don't need module rebuild
+            payload = {'batch': batch, 'build_modules': False}
+            if GAE_PROJECT == 'localhost':  # e.g., unittest case
+                batch_rebuild_participants_task(payload)
+            else:
+                task.execute('rebuild_participants_task', payload=payload, project_id=GAE_PROJECT,
+                             queue='resource-rebuild', in_seconds=15, quiet=True)
```

Comment:
Can we use the `dispatch_participant_rebuild_tasks` method instead, maybe by expanding it to include an option for defining the "build_modules" option? That way the logic for how to create a task for rebuilding a participant is better encapsulated.

Reply:
Updated.   FYI:   We're going to be deprecating all of this rebuild task dispatching when the new RDR-PDR pipeline is implemented